webpack/config.js

```diff
@@ -46,7 +50,7 @@ if (!DEBUG) {
 }
 
 const config = {
-  mode: DEBUG ? "development" : "production",
+  mode: ["development", "production"].includes(process.env.NODE_ENV) ? process.env.NODE_ENV : "none",
```

Comment:
is this switching over from using a `DEBUG` keyword in the node env to either `development` or `production`? and is that a version 16 thing or a change you made?

Reply:
I made this change due to a new error that showed up after updating webpack as part of the Node 16 upgrade. Previously, webpack set the mode to "development" if the NODE_ENV env var is set to `development` or if the WEBPACK_HOT_RELOAD env var is set to some value
https://github.com/MoveOnOrg/Spoke/blob/2e8c479ef7e4071bc903bdf882334488b50f2910/webpack/config.js#L6-L7

This works for development and production environments, but will flag a warning for our test environment. If you run the command `yarn start:test`, you will get the following error:
```
Conflicting values for 'process.env.NODE_ENV'
'"test"' !== '"development"'
```

This is because in the [env file defined for the test environment](https://github.com/MoveOnOrg/Spoke/blob/2e8c479ef7e4071bc903bdf882334488b50f2910/dev-tools/.env.test), WEBPACK_HOT_RELOAD is set to `1`, so `DEBUG` resolves to `true` and webpack will set the mode to "development". However, the NODE_ENV in the test environment's env file is set to `test`, thus leading to conflicting NODE_ENV values.

The change that I made sets the mode to "development" if NODE_ENV is set to "development", "production" if NODE_ENV is set to "production", and "none" if NODE_ENV is set to anything else to avoid this webpack warning about conflicting NODE_ENV values